### ISSUE TITLE
Is `DockFamily` still available?

### ISSUE THREAD

**Issue by khiner** [ocornut/imgui#5166]
**Version/Branch of Dear ImGui:**

Version: 1.88 WIP (tip of docking)
Branch: docking

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_sdl.cpp + imgui_impl_opengl3.cpp
Operating System: macOS

**My Issue/Question:**

Under the "API" section of https://github.com/ocornut/imgui/issues/2109#issuecomment-425974856, it says:

> The `SetNextWindowDockFamily()` API, which doesn't have a demo yet, allows you to make some windows dockable into to another set of windows. In particular, if you have multiple dockspace with tools related to a given editor or document, you may want each dockspace to use a unique family so windows from one cannot be docked into the other.

I am looking for exactly this kind of functionality, but I don't see `DockFamily` anywhere in the codebase on the `docking` branch.

Specifically, I would like to have a docked window that is restricted to contain only a restricted subset of child windows. I would like to allow some subset of related child windows to be dockable _only within_ this parent window (either as tabs or docked within it), but not docked outside it. Also, I want to disallow windows _not_ in this group from being docked into this restricted parent window.

Basically, it sounds like my needs are more or less exactly the same as this `SetNextWindowDockFamily()` description. The "restricted window group" described above would be a `DockFamily`.

Was the `DockFamily` concept/API abandoned at some point? Is there another way to achieve what I am looking for with the current docking API in `imgui_internals.h`?

Thank you!

**Comment by PathogenDavid** [comment]
`ImGuiDockFamily`, etc was renamed to `ImGuiWindowClass` in https://github.com/ocornut/imgui/commit/e194219f2e2e4b51099d45ed50017804efa9a57b

**Comment by khiner** [comment]
Thanks! Closing.
